Stockwatch/Stockwatch.WindowsApp/Form1.cs

```diff
@@ -1,10 +1,52 @@
+using Stockwatch.Business;
+
 namespace Stockwatch.WindowsApp
 {
     public partial class Form1 : Form
     {
-        public Form1()
+        private IStockSymbolPage _symbolpage;
+        public Form1(IStockSymbolPage symbolpage)
         {
+            _symbolpage = symbolpage;
             InitializeComponent();
         }
+        private void Form1_Load(object sender, EventArgs e)
+        {
+            SymbolDdown.Items.Add("Select");
```

Comment:
`SymbolDdwon` there seems to be typo in this name

Reply:
Removed